### DISCUSSION TITLE
How to temporary prevent auto install core plugin ,like nvim-treesitter\nvim-treesitter

### DISCUSSION THREAD

**Discussion by WingDust** [NvChad/NvChad#1191] [Q&A] [answered]
How to temporary prevent auto install core plugin ,like nvim-treesitter\nvim-treesitter
 I want temporay don't use nvim-treesitter\nvim-treesitter

**Comment by siduck** [comment, marked as answer]
just disable it, disable = true in a plugin's definition will uninstall it as per packer

**Reply by WingDust** [reply]
can show a code snippet for me ?

**Reply by siduck** [reply]
@WingDust https://nvchad.github.io/config/Custom%20config#enable-dashboard, you do the same with treesitter but disable = true

**Reply by WingDust** [reply]
@siduck  thanks it 's work
